### PR TITLE
Initial commit of local vs non-local TN demo

### PR DESCRIPTION
**Pull request by gsalton**
Added TN1_demo_local_vs_non-local_random_circuits notebook and associated permuted_circuit image.

First commit of TN1 demo for structured vs unstructured circuits.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/braket_features/TN1_demo_local_vs_non-local_random_circuits.ipynb [hunk 1]
```diff
@@ -0,0 +1,889 @@
+{
```
Comment (licedric) [inline examples/braket_features/TN1_demo_local_vs_non-local_random_circuits.ipynb:1]: <p>Nit: it's -&gt; its</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (licedric) [inline examples/braket_features/TN1_demo_local_vs_non-local_random_circuits.ipynb:1]: <p>Nit: In "range(num_qubits),numgates", space after comma</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (gsalton) [reply examples/braket_features/TN1_demo_local_vs_non-local_random_circuits.ipynb:1]: Fixed.

Comment (gsalton) [reply examples/braket_features/TN1_demo_local_vs_non-local_random_circuits.ipynb:1]: Fixed.

**Comment by gsalton** [comment]
Friendly bump.

**Comment by licedric** [comment]
Looks good to me. Adding @kshyatt-aws for a second opinion.

**Comment by kshyatt-aws** [comment]
LGTM. Sorry it took me so long to get to this, I somehow didn't get pinged.

**Review by kshyatt-aws — APPROVED** [review]

**Pull request merged**